### ISSUE TITLE
Delete the random junk left over in admin/

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#3165]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)| @exarkun reported|
|-|-|
|Trac ID|trac#3165|
|Type|task|
|Created|2008-04-05 00:09:51Z|
|Branch|https://github.com/twisted/twisted/tree/clean-admin-3165|

There's a bunch of things that aren't useful or relevant anymore.

<details><summary>Searchable metadata</summary>

```
trac-id__3165 3165
type__task task
reporter__exarkun exarkun
priority__low low
milestone__ 
branch__branches_clean_admin_3165 branches/clean-admin-3165
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__ 
time__1207354191000000 1207354191000000
changetime__1216211883000000 1216211883000000
version__None None
owner__ 

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="radix's avatar" src="https://avatars.githubusercontent.com/u/227068?s=50" width="50" height="50">](https://github.com/radix)<a name="note_1"></a>|@radix commented|
|-|-|

(In [24270]) Branching to 'clean-admin-3165'

**Comment by twisted-trac** [comment]
|[<img alt="radix's avatar" src="https://avatars.githubusercontent.com/u/227068?s=50" width="50" height="50">](https://github.com/radix)<a name="note_2"></a>|@radix removed owner|
|-|-|

ready to review.

For the reviewer's convenience, the changes are included here:

```
D      admin/override
D      admin/cpyrght
D      admin/clean
D      admin/release-procedure.txt
D      admin/change
D      admin/runzpkg
D      admin/xmlpp
D      admin/release-twisted
D      admin/createpackages
D      admin/process-docs
D      admin/make-deb
D      admin/echo.py
D      admin/lowertags
D      admin/notify-on-release.txt
```

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_3"></a>|@exarkun set owner to @radix|
|-|-|

looks good, please merge.

**Comment by twisted-trac** [comment]
|[<img alt="radix's avatar" src="https://avatars.githubusercontent.com/u/227068?s=50" width="50" height="50">](https://github.com/radix)<a name="note_4"></a>|@radix set status to `closed`|
|-|-|

(In [24278]) Merge clean-admin-3165

Author: radix
Reviewer: exarkun
Fixes: [#3165](https://github.com/twisted/twisted/issues/3165)

Delete most of the files in admin/, because they are now unused.

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
